### PR TITLE
Navbar

### LINKED ISSUE #3: Nav bar

### PR DESCRIPTION
**Pull request by SMALLASIAN**
close #3 
- [x] in progress
- [x] testing
- [x] review

### CODE REVIEW

**Comment by SMALLASIAN** [comment]
Screenshot 2022-12-14 at 17 54 31
Screenshot 2022-12-14 at 17 54 45

**Comment by SMALLASIAN** [comment]
Screenshot 2023-01-28 at 17 10 22
Screenshot 2023-01-28 at 17 10 37
Screenshot 2023-01-28 at 17 11 00

**Comment by SMALLASIAN** [comment]
Screenshot 2023-01-28 at 17 11 55

File: app/views/navbar/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<style>
+  .dropdown:hover .dropdown-menu{
+      display: block;
+  }
+  .dropdown-menu{
+      margin-top: 0;
+  }
+</style>
+  <script>
```
Comment (awiktorowicz) [inline app/views/navbar/_navbar.html.erb:9]: JavaScript and Style should not be defined in a View. You could include it in the view by adding a similar script `<%= javascript_include_tag "[Name_of_the_js_file]" %>` .  This way, the script will only be loaded when the view is rendered and not on every page. It is also a good practice to minify and compress the javascript files for the production environment.

**Review by awiktorowicz — CHANGES_REQUESTED** [review]

File: app/helpers/team_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module TeamHelper
```
Comment (awiktorowicz) [inline app/helpers/team_helper.rb:1]: Do we need it?

File: app/helpers/recruitment_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module RecruitmentHelper
```
Comment (awiktorowicz) [inline app/helpers/recruitment_helper.rb:1]: Same as above

File: app/helpers/project_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module ProjectHelper
```
Comment (awiktorowicz) [inline app/helpers/project_helper.rb:1]: Same as above

File: app/helpers/idea_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module IdeaHelper
```
Comment (awiktorowicz) [inline app/helpers/idea_helper.rb:1]: Same as above

File: app/helpers/home_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
```
Comment (awiktorowicz) [inline app/helpers/home_helper.rb:1]: Same as above

File: app/helpers/contact_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module ContactHelper
```
Comment (awiktorowicz) [inline app/helpers/contact_helper.rb:1]: Same as above

File: app/helpers/achievement_helper.rb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+module AchievementHelper
```
Comment (awiktorowicz) [inline app/helpers/achievement_helper.rb:1]: Same as above

**Review by awiktorowicz — CHANGES_REQUESTED** [review]

**Pull request merged**